src/new-components/banner/banner.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react'
+import { Box } from '../box'
+import { Columns, Column } from '../columns'
+
+import styles from './banner.module.css'
+
+export type BannerTone = 'info' | 'promotion'
+
+type BannerProps = {
+    id?: string
+    /**
+     * The tone of the Banner. Affects the background color and the outline.
+     */
+    tone: BannerTone
+    /**
+     * The icon that should be added inside the Banner.
+     */
+    icon: React.ReactChild
+    /**
+     * The title to be displayed at the top of the Banner.
+     */
+    title: string
+    /**
+     * An optional description to be displayed inside the Banner.
+     */
+    description?: string
```

Comment:
I tend to prefer to declare these as type `React.ReactNode` instead.

```suggestion
    /**
     * The title to be displayed at the top of the Banner.
     */
    title: React.ReactNode
    /**
     * An optional description to be displayed inside the Banner.
     */
    description?: React.ReactNode
```

It enables passing rich text, for instance, containing links or bold or emphasized text:

```jsx
<Banner
  title={<>This is really <strong>important</strong></>}
  description={<>If you're curious, <a href="#">learn more</a>.</>}
/>
```

Reply:
This makes a lot of sense! 😅 I added a unit test and also an example in the stories. Which lead me to:

<img width="738" alt="Screenshot 2023-01-26 at 16 07 15" src="https://user-images.githubusercontent.com/1509326/214871562-0dbb91f7-1964-4b8f-870c-190d666e6f2b.png">

<img width="714" alt="Screenshot 2023-01-26 at 16 07 19" src="https://user-images.githubusercontent.com/1509326/214871594-43ac58ea-6b9e-4692-84ba-655c6e7a3630.png">

Is this (colors for href) something that I should already ask Design team about or should we just move forward since we don't have any use cases (in todoist-web) at the moment?

Reply:
What if you use `<TextLink />` instead of `<a />`? Though it probably won't get the right color either, because we do not have colors for dark mode here in Reactist. Unless you also provide the necessary CSS variable override as you do with the rest of the colors for dark mode in this story.